spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/BaseCatalog.java

```diff
@@ -35,7 +35,9 @@ abstract class BaseCatalog
         ProcedureCatalog,
         SupportsNamespaces,
         HasIcebergCatalog,
-        SupportsFunctions {
+        SupportsFunctions,
+        org.apache.spark.sql.connector.catalog.ViewCatalog,
```

Comment:
Why does this need to be fully qualified?

Reply:
not needed anymore